plugins/gatsby-remark-check-links-numberless/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "gatsby-remark-check-links-numberless",
+  "version": "1.0.0",
+  "description": "Transformed links checker",
+  "main": "index.js",
+  "author": "Nilufar Bava <nilubava@gmail.com>",
+  "license": "ISC",
```

Comment:
That might not be an appropriate licence for software from us. You might want to ask around.